packages/manager/apps/dedicated/client/app/dedicatedCloud/details/dedicatedCloud.routing.js

```diff
@@ -262,6 +262,14 @@ export default /* @ngInject */ ($stateProvider) => {
             datacenterId,
           },
         ),
+      resizeNsxEdge: /* @ngInject */ ($state) => (datacenterId) => {
```

Comment:
```suggestion
      goToResizeNsxEdge: /* @ngInject */ ($state) => (datacenterId) => {
```
To be compliant with functions using `$state.go(`